internal/app/machined/pkg/adapters/network/nftables_rule.go

```diff
@@ -91,7 +91,7 @@ func (set NfTablesSet) SetElements() []nftables.SetElement {
 
 		for _, p := range set.Ports {
 			from := binaryutil.BigEndian.PutUint16(p[0])
-			to := binaryutil.BigEndian.PutUint16(p[1])
+			to := binaryutil.BigEndian.PutUint16(p[1] + 1)
```

Comment:
was this a bug?

Reply:
yes, I will separate it out from the machine config changes when I finalize it.

It was even visible in the test lol